plugins/generator-1.15.2/forge-1.15.2/templates/json/tag.json.ftl

```diff
@@ -15,6 +15,11 @@
             "${generator.getResourceLocationForModElement(value)}"
             <#if value?has_next>,</#if>
           </#list>
+      <#elseif data.type == "Entities">
+          <#list data.entities as value>
+            "${generator.getResourceLocationForModElement(value)}"

```

Comment:
This is not the right approach for entities, you need to use a 3-rd mapping table of entities for vanilla entities and registry name for custom entities.

Same for 1.16.4.

Reply:
On the [Vanilla wiki](https://minecraft.gamepedia.com/Tag#JSON_format), it is written that all tags are registered the same in `.json` files.

Reply:
What does this do with the mapping system? You need to change the mapping system, not the file format, I did not say that

Reply:
Oh, and how do I do that? What mapping system do I need to use?

Reply:
Entity, but 3-rd mapping table (index 2) which defines Minecraft registry entity names.

Reply:
See https://github.com/MCreator/MCreator/blob/master/plugins/generator-1.16.4/forge-1.16.4/mappings/entities.yaml, 1.15.2 is missing this table currently

Reply:
@MysteriousWolf can add for 1.15.2 I think

Reply:
Pull master into PR, they are now there for 1.15.2 too

Reply:
Damn, I was implementing them...